compute_endpoint/globus_compute_endpoint/cli.py

```diff
@@ -242,6 +242,9 @@ def version_command():
         "https://docs.globus.org/api/auth/developer-guide/#authentication-policies."
     ),
 )
+@click.option(
+    "--subscription-uuid", hidden=True, help="Associate endpoint with a subscription"
```

Comment:
To be consistent with the Globus SDK, etc., we should use `-id` instead of `-uuid`.